components/brave_wallet_ui/page/screens/send/components/standard-button/standard-button.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// Assets
+import InfoIcon from '../../../../../assets/svg-icons/info-icon.svg'
+
+// Styled Components
+import { Button, ErrorIcon } from './standard-button.style'
+
+export interface StandardButtonStyleProps {
+  buttonType?: 'primary' | 'secondary' | 'danger'
+  buttonWidth?: 'dynamic' | 'full'
+  disabled?: boolean
+  marginRight?: number
+  hasError?: boolean
+}
```

Comment:
Let's move this to `./standard-button.style`

Reply:
Moved!